app/pages/cif/create-project.tsx

```diff
@@ -62,24 +47,18 @@ export function CreateProject({
     await updateFormChangeMutation(preloadedQuery.environment, variables);
   };
 
-  const onValueChanged = async (change) => {
-    const { formData } = change;
-    await storeResult(formData);
-  };
-
-  /* Uncomment this block when the applyChangeFromComponent function is ready to be used.
-   Currenly commented out so the linter won't complain about the unused function.
-
   // The applyChangeFromComponent function will require this page to be aware of the state of the newFormData object
   const formChangeData = query.formChange.newFormData;
 
   // A function to be called by individual components making changes to the overall form_change data
-  const applyChangeFromComponent = (changeObject: any)  => {
+  const applyChangesFromComponent = (changeObject: any) => {
```

Comment:
Is this needed if we have a 1-to-1 mapping between a jsonschema form and a form_change record? Or is that not the case?

Reply:
The idea put forward when doing the exploration was to have that one-to-one association between a form_change and a jsonschema form. I don't know how high fidelity those designs are, and whether we need to stick the same ordering of fields, especially if that means adding features in the code to support a different layout (unless that is an optimal layout that is supported by user research). If needed, I'd argue that the design needs to be updated to ensure that the form fields that will end up in the same form_change are located together

Reply:
I'm all up for that, happy to refactor - and remove code :) 
Since we maintain a theme for rjsf now, it'll be probably a lot easier to write a few custom object types and renderers to match the wireframes

Reply:
Does this mean each `fieldset` in the wireframe is not a separate component & we will have one component that renders all those fields with one JsonSchemaForm?

Reply:
The <fieldset> can be part of the form schema (with an object template) and rendered by rjsf instead of wrapping an entire block

Reply:
Gotcha. One "create-project" component, one form schema, many fieldsets rendered by rjsf

Reply:
one form schema per `form_change` database record, in this case, one for the project, one per contact, etc.
Everything will be wrapped in one revision when it's implemented as part of #69